.github/workflows/macos-linux-windows-pixi.yml

```diff
@@ -0,0 +1,95 @@
+name: CI - MacOS/Linux/Windows via Pixi
+
+on:
+  push:
+    paths-ignore:
+      - .gitlab-ci.yml
+      - .gitignore
+      - '*.md'
+      - CITATION.*
+      - LICENSE
+      - colcon.pkg
+      - .pre-commit-config.yaml
+      - CHANGELOG.md
+      - development/*.md
+  pull_request:
+    paths-ignore:
+      - .gitlab-ci.yml
+      - .gitignore
+      - '*.md'
+      - CITATION.*
+      - LICENSE
+      - colcon.pkg
+      - .pre-commit-config.yaml
+      - CHANGELOG.md
+      - development/*.md
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  nanoeigenpy-pixi:
+    name: ${{ matrix.os }} - Env ${{ matrix.environment }} ${{ matrix.build_type }} ${{ matrix.cxx_options }}
+    runs-on: ${{ matrix.os }}
+    env:
+      CCACHE_BASEDIR: "${GITHUB_WORKSPACE}"
+      CCACHE_DIR: "${GITHUB_WORKSPACE}/.ccache"
+      CCACHE_COMPRESS: true
+      CCACHE_COMPRESSLEVEL: 6
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-13]
+        environment: [default, python-oldest]
+        build_type: [Release, Debug]
+        cxx_options: ['', '-mavx2']
+        exclude:
+          - os: macos-latest
+            cxx_options: '-mavx2'
+          - os: macos-13
+            cxx_options: '-mavx2'
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+
+    - uses: actions/cache@v4
+      with:
+        path: .ccache
+        key: ccache-macos-linux-windows-pixi-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.cxx_options }}-${{ matrix.environment }}-${{ github.sha }}
+        restore-keys: ccache-macos-linux-windows-pixi-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.cxx_options }}-${{ matrix.environment }}-
+
+    - uses: prefix-dev/setup-pixi@v0.8.3
+      with:
+        cache: true
+        environments: ${{ matrix.environment }}
+
+    - name: Build nanoeigenpy [MacOS/Linux/Windows]
+      env:
+        CMAKE_BUILD_PARALLEL_LEVEL: 2
+        NANOEIGENPY_BUILD_TYPE: ${{ matrix.build_type }}
+        NANOEIGENPY_CXX_FLAGS: ${{ matrix.cxx_options }}
+      run: |
+        pixi run -e ${{ matrix.environment }} build
+
+    - name: Test nanoeigenpy [MacOS/Linux/Windows]
+      if: ${{ ! (contains(matrix.os, 'windows') && matrix.build_type == 'Debug') }}
```

Comment:
Why this if ? You didn't set windows as os right now.

Reply:
I just copied the pixi file from coal actually 😅

Reply:
We have windows now, so it's ok right?

Reply:
Yep, but why we doesn't when to run tests in this case ?

Reply:
Oops